packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -0,0 +1,193 @@
+import Edit from '@spectrum-icons/workflow/Edit';
+import {Item} from '@react-stately/collections';
+import RegionSelect from '@spectrum-icons/workflow/RegionSelect';
+import Select from '@spectrum-icons/workflow/Select';
+```
+
+# ActionGroup
+
+<p>{docs.exports.ActionGroup.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionGroup']} />
+
+## Example
+
+```tsx example
+<ActionGroup>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+## Content
+
+ActionGroups accept as list of `children`, each with a `uniqueKey` prop.
+
+It is required that the `children` are Item components from the `@react-stately/collections` library.
+In order to identify children when a user takes an action on a item, each child needs to have a `uniqueKey` prop.
+See [Events](#events) for more information.
+
+## Value
+
+ActionGroups are uncontrolled by default. 
+
+Use `defaultSelectedKeys` to provide a default set of selected items. The value of the selected keys must match the `uniqueKey` prop of the items.
+
+```tsx example
+<ActionGroup defaultSelectedKeys={['delete']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+The `selectedKeys` prop can be used to make the selected state controlled.
+
+```tsx example
+<ActionGroup selectedKeys={['delete']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+### Accessibility
+
+If no visible label is provided (e.g. an icon only) to each item, an alternative text label must be provided to identify the control for accessibility.
+This should be added using the `aria-label` prop to each Item.
+
+```tsx example
+<ActionGroup>
+  <Item uniqueKey="brush" aria-label="Brush"><Brush /></Item>
+  <Item uniqueKey="select" aria-label="Select"><Select /></Item>
+  <Item uniqueKey="regionSelect" aria-label="Select Region"><RegionSelect /></Item>
+</ActionGroup>
+```
+
+### Internationalization
+
+In order to internationalize an ActionGroup, the strings of all child items should be localized.
+
+## Events
+
+Use the `onSelectionChange` prop as a callback to handle press events on items.
+The `uniqueKey` prop from the selected item will be passed into the callback.
+
+The following example uses the `onSelectionChange` prop to update a string stored in React state.
+
+```tsx example
+function Example() {
+  let [lastKey, setLastKey] = React.useState('');
+  return (
+    <div>
+      <ActionGroup onSelectionChange={setLastKey}>
+        <Item uniqueKey="add">Add</Item>
+        <Item uniqueKey="delete">Delete</Item>
+        <Item uniqueKey="edit">Edit</Item>
+      </ActionGroup>
+      <p>Last selected: <span>{lastKey}</span></p>
+    </div>
+  );
+}
+```
+
+For multiple selections, use the `selectionMode` prop with the `multiple` value.
```

Comment:
output is currently `Selected: deleteeditadd`
maybe to do something like:

```
    let [selectedKeys, setSelectedKeys] = React.useState(new Set()); // start value to be just empty Set, since onSelectionChange is returning Set of values or empty Set for non seleceted
    return (
      <div>
        <ActionGroup selectionMode="multiple" onSelectionChange={setSelectedKeys}> // could be just setSelectedKeys
          <Item uniqueKey="add">Add</Item>
          <Item uniqueKey="delete">Delete</Item>
          <Item uniqueKey="edit">Edit</Item>
        </ActionGroup>
        <p>Selected: {[...selectedKeys].toString()}</p> // no need for spans
      </div>
```